plugin/pkg/admission/security/podsecuritypolicy/admission.go

```diff
@@ -354,11 +355,19 @@ func isAuthorizedForPolicy(user, sa user.Info, namespace, policyName string, aut
 }
 
 // authorizedForPolicy returns true if info is authorized to perform the "use" verb on the policy resource.
+// TODO: check against only the policy group when PSP will be completely moved out of the extensions
```

Comment:
@liggitt @tallclair For how long we need to support PSP in extenstions API group?